x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/action/saml/TransportSamlInvalidateSessionActionTests.java

```diff
@@ -355,7 +373,7 @@ public void testInvalidateCorrectTokensFromLogoutRequest() throws Exception {
         action.doExecute(mock(Task.class), request, future);
         final SamlInvalidateSessionResponse response = future.get();
         assertThat(response, notNullValue());
-        assertThat(response.getCount(), equalTo(2));
+        assertThat(response.getCount(), equalTo(4));
```

Comment:
This is something I honestly did not foresee.
The SAML invalidate session endpoint returns the number of token **pairs** found to be invalidated, but my change changes it to return the number of tokens found to be invalidated. That is, an access and refresh token pair is now counted as 2 tokens. I don't think this change is significant.

Reply:
I've checked the Kibana side and it seems that it does not depend on the number of invalidated tokens. So I think we are okay with this change.

https://github.com/elastic/kibana/blob/8683621a65620fc079b52ad3e963f639bf4f3481/x-pack/plugins/security/server/authentication/providers/saml.ts#L641